source/ddbus/meson.build

```diff
@@ -0,0 +1,33 @@
+lib_src = [
+    'attributes.d',
+    'bus.d',
+    'c_lib.d',
+    'conv.d',
+    'exception.d',
+    'package.d',
+    'router.d',
+    'simple.d',
+    'thin.d',
+    'util.d',
+]
+
+ddbus_lib = library(
+    'ddbus',
+    lib_src,
+    install: true,
+    include_directories: src_inc,
+    link_args: ['-shared'],
```

Comment:
This doesn't need to be an array.

Reply:
Ah right, forgot to change it after removing the dynamic linking thingie for Phobos.

Reply:
does it hurt though? When adding more args in the future this avoids issues

Reply:
See our discussion above. The downstream shouldn't be this opinionated. The default configuration should work in most case. If the user wants to change it, he can change it (ex. by passing `DFLAGS`).

In this particular case, adding this flag would for ex. break the project for default GCC installations. On top of that, there isn't anything that the build system or the user can do about it because the flag is being explicitly added. To fix it the user would need to remove the flag from the meson file, which is just not how build systems should work.

Reply:
I think @WebFreak001 referred to this being an array and not linking phobos dynamically

Reply:
Ah right, not really. It is just the standard syntax for meson files. Otherwise, you would stick `[]` almost everywhere as most fields support receiving arrays.

Reply:
Done